sklearn/utils/tests/test_extmath.py

```diff
@@ -37,21 +37,27 @@
     svd_flip,
     weighted_mode,
 )
-from sklearn.utils.fixes import _mode
+from sklearn.utils.fixes import (
+    COO_CONTAINERS,
+    CSC_CONTAINERS,
+    CSR_CONTAINERS,
+    DOK_CONTAINERS,
+    LIL_CONTAINERS,
+    _mode,
+)
 
 
-def test_density():
+@pytest.mark.parametrize(
+    "sparse_container",
+    [*COO_CONTAINERS, *CSC_CONTAINERS, *CSR_CONTAINERS, *LIL_CONTAINERS],
```

Comment:
```suggestion
    COO_CONTAINERS + CSC_CONTAINERS + CSR_CONTAINERS + LIL_CONTAINERS,
```

Reply:
@glemaitre Is there a reason why we are preferring the suggestion to using *? I am asking because I considered both to be okay.

Reply:
It was the style used in Julien's first PR and the main reason why I'm sticking to it. But if we merged some with this style, there is no need to comeback on it.

On a subjective point of view, I find the concatenation shorter to write and more straightforward to understand than an unpacking within a new list. I would rather limit the unpacking in function call only. But as said this is more of a personal preference.

Reply:
Thanks for clarifying. I think the concatenation seems fine so we can proceed with this style.